### PR TITLE
TWL-374 Added TF resource for workspace

### PR DESCRIPTION
**Pull request by mckornfield**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ require (
 	github.com/agext/levenshtein v1.2.1
 	github.com/apparentlymart/go-cidr v0.0.0-20170616213631-2bd8b58cf427
 	github.com/apparentlymart/go-textseg v1.0.0
-	github.com/appian/twilio-go v0.0.0-20190630185733-fe05957cdaf8
+	github.com/appian/twilio-go v0.0.0-20191021162026-dc5703031a9b
```
Comment (mckornfield) [inline go.mod]: we made this change manually, yay git

File: plugin/providers/twilio/resource_twilio_workspace.go [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+	}
+	d.Set("friendly_name", workspace.FriendlyName)
+	d.Set("date_created", workspace.DateCreated)
+	d.Set("date_updated", workspace.DateUpdated)
+	d.Set("multi_task_enabled", workspace.MultiTaskEnabled)
+	d.Set("sid", workspace.Sid)
+	return nil
+}
+
+func resourceTwilioWorkspaceUpdate(d *schema.ResourceData, meta interface{}) error {
```
[the remark this answers is not in the document]
Comment (samratjha96) [reply plugin/providers/twilio/resource_twilio_workspace.go]: Yep. Just that. Lol. And hopefully the plan shows up with a `~` for the field you changed and that the new friendly name actually shows up on the twilio console

[the remark this answers is not in the document]
Comment (mckornfield) [reply plugin/providers/twilio/resource_twilio_workspace.go]: I tested via changing, seems to work, though application kept saying it needed to change :)

File: go.mod (hunk 1 above)
Comment (samratjha96) [reply go.mod]: I don't quite get this change? Is this so we can use the `workspace.go` implementation that merged into `twilio-go` repo master? Anyway to have go automatically point to what's in master? Or we are gonna have to make this change multiple times whenever we add a go implementation right?

File: plugin/providers/twilio/resource_twilio_workspace.go (hunk 2 above)
[the remark this answers is not in the document]
Comment (mckornfield) [reply plugin/providers/twilio/resource_twilio_workspace.go]: Isaac verified via frontend

File: go.mod (hunk 1 above)
Comment (mckornfield) [reply go.mod]: we have to make the change each team. the number after 0.0.0 is the commit time, and the last part is a twelve character sha

Comment (mckornfield) [reply go.mod]: for example, on line ten, the bgentry/go-netrc line matches up to this commit https://github.com/bgentry/go-netrc/commit/9fd32a8b3d3d3f9d43c341bfe098430e07609480

Comment (mckornfield) [reply go.mod]: yes once you make the change

**Comment by samratjha96** [comment]
LGTM 🚀

**Pull request merged**